src/Controller/DeveloperAppListBuilderForDeveloper.php

```diff
@@ -76,7 +82,8 @@ public static function create(ContainerInterface $container) {
       $container->get('entity.manager')->getStorage($entity_type->id()),
       $container->get('entity.manager'),
       $container->get('renderer'),
-      $container->get('current_user')
+      $container->get('current_user'),
+      $container->get('request_stack')
```

Comment:
Because of JS. As I see it uses the current URI and app ID to make difference between toggle buttons. I've replaced the global calling of the service with dependency injection.